Documentation/Books/AQL/Functions/Miscellaneous.md

```diff
@@ -41,6 +41,37 @@ For conditional evaluation, check out the
 Database functions
 ------------------
 
+CHECK_DOCUMENT()
```

Comment:
should be heading level 3, instead of 2 - see below screnshoot

![screenshot 2019-01-04 13 56 42](https://user-images.githubusercontent.com/31849787/50689170-d46a8e00-1028-11e9-8b0b-39f4d7ca7f79.png)

Doc build is green - preview: http://web01.arangodb.biz/bug-fix/add-aql-check-document/AQL/Functions/Miscellaneous.html